wbobeirne/joule-extension
# Add new screen to display wallet balance sources

Closes #31 (already closed)
I originally intended to include this feature with the on-chain sending but I knew this code change was going to be too big, so here's a separate PR.

### Description

This is my attempt at trying to answer the question "Why are my balances not adding up?" which many users have asked. The typical reason is simply because you opened a channel of a small size but LND used a UTXO which is much larger, so until the channel is opened you cannot spend the full amount of the UTXO. 

This screen also surfaces some other useful information such as:
- I have a channel open with an offline node, when were they last online?
- How long will it take for my pending balance to become available?
- What UTXOs do I have in my wallet?
- When will this force closed channel be confirmed?

### Steps to Test

1. View the Balances screen and confirm it looks accurate for your node
2. Open & Close channels to see the pending balances
3. Send & Receive on-chain coins to see your UTXOs get updated

### Screenshots

<img src="https://user-images.githubusercontent.com/1356600/53295415-fb437600-37c8-11e9-8548-121f1a2f4043.gif" width="500" />
